src/app/storage/query/utils.ts

```diff
@@ -0,0 +1,108 @@
+import { Method } from 'axios';
```

Comment:
other "changes since last view" in this file are cause by correcting the nested type redundancy pointed out by @joneubank here:
> https://github.com/supercluster-covid-data-portal/api/pull/5#discussion_r748542979